apikit-tools/apikit-scaffolder/src/main/java/org/mule/tools/apikit/output/scopes/FlowScope.java

```diff
@@ -30,11 +26,7 @@ public FlowScope(Element mule, String exceptionStrategyRef, API api, String conf
 
         if (httpListenerConfigRef != null)
```

Comment:
there is no need to have this "if" anymore. The method will always receive a listener config.